recipes/intel-ipsec-mb/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import tools, ConanFile, AutoToolsBuildEnvironment
+
+required_conan_version = ">=1.39.0"
+
+class IntelIpSecMbConan(ConanFile):
+    name = "intel-ipsec-mb"
+    description = "Intel(R) Multi-Buffer Crypto for IPSec"
+    license = "BSD 3-Clause"
+    homepage = "https://github.com/intel/intel-ipsec-mb"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("intel", "IPSec", "IPSec_MB")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        print(self.conan_data["sources"][self.version])
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        return self._autotools
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        if not tools.which("make"):
+            self.build_requires("make/4.2.1")
+        if not tools.which("nasm"):
+            self.build_requires("nasm/2.15.05")
```

Comment:
Just add it as a build requirement and not depend on a possibly faulty/old locally installed nasm.
```suggestion
        self.build_requires("nasm/2.15.05")
```